app/controllers/home_controller.rb

```diff
@@ -5,7 +5,23 @@ def type;end
 
   def faqs; end
 
-  def profile_psychologist; end
+  def profile_psychologist; 
+    
+  end
+
+  def psychologist_List
+    @Psychologist = Psychologist.order(params[:value])
```

Comment:
Same for instance variable names, usually all lowercase. I know is a small thing, but code style is important. :D